Makefile

```diff
@@ -244,4 +244,56 @@ test-sim-after-import:
 	go test -mod=readonly -tags=$(BUILD_MAINNET) $(APP_DIR) -run=TestAppSimulationAfterImport -Enabled=true \
 		-NumBlocks=50 -BlockSize=100 -Commit=true -Seed=99 -Period=5 -v -timeout 10m
 
-test-sims: test-sim-fullapp test-sim-nondeterminism test-sim-import-export test-sim-after-import
\ No newline at end of file
+test-sims: test-sim-fullapp test-sim-nondeterminism test-sim-import-export test-sim-after-import
+
+proto-gen:
+	@./script/protocgen.sh
```

Comment:
extremely small nitpick: I prefer to see the command that is executing; do you mind not using `@`?